packages/react-app/src/hooks/use-toggle-theme.ts

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useEffect, useState } from 'react';
+import { DEFAULT_THEME } from 'src/constants';
+import { customDarkTheme } from 'src/styles/dark-theme';
+import { customLightTheme } from 'src/styles/light-theme';
+
+export const useToggleTheme = () => {
+  // const { currentTheme, setCurrentTheme } = useThemeContext();
```

Comment:
Sure we dont want to use the context here ?

Reply:
yah my first thgout was use there, but useThemeContext use that hook, so i think its better the hook manage the changes and the states of variable.